modules/pel/peltool/peltool.py

```diff
@@ -389,7 +389,7 @@ def parsePELSummary(stream: DataStream, config: Config):
     ret, ph = generatePH(stream, out)
     if ret is False:
         return "", ""
-    eid = ph.pLID
+    eid = ph.lEID
```

Comment:
Please avoid mixing multiple changes in a single commit.

Each changes should be pushed separately, with a commit message header that clearly states what you are trying to achieve in a simple sentence. The commit message body should explain why you are making these changes and optionally, how you are implementing them.

In the current commit messages, it's unclear why the changes are being made. For instance, "option names renamed" doesn't provide enough context.